diff -Naur mod_auth_pgsql-0.9.10-p/mod_auth_pgsql.c mod_auth_pgsql-0.9.10/mod_auth_pgsql.c
--- mod_auth_pgsql-0.9.10-p/mod_auth_pgsql.c	Thu Oct 11 12:47:14 2001
+++ mod_auth_pgsql-0.9.10/mod_auth_pgsql.c	Wed Oct 24 15:59:09 2001
@@ -96,7 +96,7 @@
 #include "ap_alloc.h"
 #include <libpq-fe.h>
 
-#define 	AUTH_PGSQL_VERSION		"0.9.9"
+#define 	AUTH_PGSQL_VERSION		"0.9.10"
 
 /* We need Apache 1.3.1 */ 
 #if (MODULE_MAGIC_NUMBER >= 19980713)
